libraries/conf_helpers.rb

```diff
@@ -27,35 +39,74 @@ def self.read_config(conf_path)
     end
 
     def self.parse_config(conf_body)
-      current_stanza = 'default'
+      current_section = 'default'
       current_config = {}
 
       conf_body.each_line do |ln|
         case ln
         when /^\s*\[([^#]+)\]\s*/
-          current_stanza = Regexp.last_match[1]
+          current_section = Regexp.last_match[1]
         when /^\s*([^#]+?)\s*=\s*([^#]+?)\s*(?:#.*)?$/
-          (current_config[current_stanza] ||= {})[Regexp.last_match[1]] = Regexp.last_match[2]
+          (current_config[current_section] ||= {})[Regexp.last_match[1]] = Regexp.last_match[2]
         end
       end
 
       current_config
     end
 
     def self.merge_config(current_config, desired_config)
```

Comment:
It took me a bit to understand where the filtering happens, but I found it here, and as I think through it, should this method be broken up? It seems like this one method does 3 different functions that might be better reallocated elsewhere?

1) Load and pull forward data from disk (which might be able to be one of many levels of procs)
2) Filter nulls (Which probably should be part of evaluate_config so decisions can be made based on the resultant map? )
3) Render the map to text to be written to disk (which would this be better as an ERB template? )